src/components/ArticleCards.jsx

```diff
@@ -1,12 +1,21 @@
 import { Link } from 'react-router-dom';
 
 export default function ArticleCards ({ articles }) {
+    const getTopic = (topic) => {
+    switch (topic) {
+        case 'coding': return 'coding';
+        case 'cooking': return 'cooking';
+        case 'football': return 'football';
+        default: return '';
+    }
+};
```

Comment:
I'm struggling to see the benefit of this, it looks like this function simply returns the same string as the input...